document how to enable dev on both Daml and Canton (#14838)
digital-asset/daml: docs/source/daml/reference/interfaces.rst

implementation, so other developers can write applications in terms of the
interface instead of the concrete template.

Configuration
*************

> In order to be able to test this experimental feature, you need to enable

Reviewer: ```suggestion
In order to test this experimental feature, you need to enable
```

> the dev mode on both Daml and Canton.

Reviewer 2: ```suggestion
the dev mode for both the Daml compiler and Canton.
```

> For Daml, you need to add in ``daml.yaml`` the following line:

Reviewer 2: ```suggestion
For the Daml compiler, add the following line to your `daml.yaml` file:
```

> For Daml, you need to add in ``daml.yaml`` the following line:

Reviewer: ```suggestion
To enable dev mode for Daml, add ``daml.yaml`` the following line:
```
Perhaps this is clear to a more experienced programmer, but for completeness' sake: do you add daml.yaml to the end of the line, or where exactly?

Author: So what I meant is that : there is an existing file named `daml.yaml` and this file we need a code line.

.. code-block:: none

   build-options: [--target=1.dev]

> For Canton, you need to follow [this documentation](https://www.canton.io/docs/dev/user-manual/usermanual/FAQ.html#how-do-i-enable-unsupported-features)

Reviewer: ```suggestion
To enable dev mode for Canton, follow the instructions in [this documentation](https://www.canton.io/docs/dev/user-manual/usermanual/FAQ.html#how-do-i-enable-unsupported-features)
```

> To enable dev mode for Canton, follow the instructions in [this documentation](https://www.canton.io/docs/dev/user-manual/usermanual/FAQ.html#how-do-i-enable-unsupported-features)

Reviewer 2: Can you add an anchor in the Canton repo and use `:ref` to link to that to ensure that we get a compile error if this link goes stale and so that it links to docs.daml.com?

Author: ok thanks
